### DISCUSSION TITLE
YOLO

### DISCUSSION THREAD

**Discussion by FootballPoolsHazel** [community/community#18568] [Other Feature Feedback, Questions, & Ideas]
I'd like to report a bug, please.
On my profile I have unlocked the yolo achievement on a private repository, however, when I click on the PR in question, it shows a PR that was approved by another member of staff. That person then left a comment "looks good" and then I merged. Yet, for some reason, I got the yolo badge from this.
This isn't great, as merging without a PR is a big no-no in my organisation where I work, so this badge sends the wrong message to my coworkers.

**Comment by herkura** [comment]
> I'd like to report a bug, please. On my profile I have unlocked the yolo achievement on a private repository, however, when I click on the PR in question, it shows a PR that was approved by another member of staff. That person then left a comment "looks good" and then I merged. Yet, for some reason, I got the yolo badge from this. This isn't great, as merging without a PR is a big no-no in my organisation where I work, so this badge sends the wrong message to my coworkers.

@FootballPoolsHazel How to unlock YOLO Badge?

**Reply by sanjay-kv** [reply]
@herkura If it's in your repository add a third-party reviewer for the PR (otherwise the task did not count) under the reviewer tab and go for the pull request.

**Reply by herkura** [reply]
@sanjay-kv thanks

**Reply by Dnkfff** [reply]
How to get YOLO patch for discord PR?

**Comment by jsirianni** [comment]
Same, would prefer to just remove this. Its for a critical repo at that.

**Comment by gumusoyku** [comment]
### I got the YOLO badge for the pull request that I was the author of!

We have this codeowners file which I am also a part of. Opened a PR, two of my team members approved and merged the PR. Got this shame badge. There is no way that I could approve my own PR!
+1 for this badge sending the wrong message to the teammates.

**Reply by FootballPoolsHazel** [reply]
In my case, I wasn't a code owner but I was an admin on the repository. It's possible the bug is related to elevated permissions.

**Comment by ningfdx** [comment]
You Only Look Once?

**Comment by sonya** [comment]
I had the same exact experience as @FootballPoolsHazel: navigated to the PR mentioned in the badge, saw that the PR was opened by me, approved by another member, and merged by me 2 minutes after the approval. I am also an admin on the repo. Other PRs in the same repo with an approver did not seem to trigger this badge, however.

From several conversations about the YOLO badge I see that many people find the messaging problematic; at the same time there are also a lot of questions on how to get this achievement, which suggests to me that the YOLO badge is unreliable, and seeing it on someone's profile does not tell you anything about that developer.

**Comment by vietnamesekid** [comment]
Hj @FootballPoolsHazel, you can submit a PR in a public repository => and add a friend as a reviewer but merge it without your friend approving the PR =))

For more info, please click [here](https://github.com/Schweinepriester/github-profile-achievements).

**Reply by FootballPoolsHazel** [reply]
Apologies, I don't think you read the question.
